tests/unit/test_attribute_configuration_collection.py

```diff
@@ -0,0 +1,335 @@
+from unittest import TestCase
+
+import responses
+
+from tamr_unify_client import Client
+from tamr_unify_client.auth import UsernamePasswordAuth
+from tamr_unify_client.models.attribute_configuration.collection import (
+    AttributeConfigurationCollection,
+)
+
+
+class TestAttributeConfigurationsCollection(TestCase):
```

Comment:
nit: Class name should be `TestAttributeConfigurationCollection` (singular Configuration)